api/app/Models/Pool.php

```diff
@@ -63,4 +67,32 @@ public function poolCandidates(): HasMany
     {
         return $this->hasMany(PoolCandidate::class);
     }
+
+    /* accessor to obtain Advertisement Status, depends on two variables regarding published and expiry */
+    public function getAdvertisementStatusAccessorAttribute()
+    {
+        date_default_timezone_set('America/Vancouver');
```

Comment:
It may be better to set this [`api/config/app.php`](https://github.com/GCTC-NTGC/gc-digital-talent/blob/c7e322050f265495d29a9a4217e4d8a4f91e31e0/api/config/app.php#L71) so that we have consistent times across the entire application.

Reply:
I'm hesitant about doing something like that without it being set in stone by the team, definitely something to discuss
[Rest of feedback](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3225/commits/90473b7b2908f329be4a4565160bdfec57f6b803)